style.css

```diff
@@ -529,3 +529,212 @@ p {
 a {
   text-decoration: none;
 }
+
+.footer {
+  background: #e5e5e5;
+}
+
+.containers {
```

Comment:
- [ ] KIndly make sure the hamburger menu is not visible on the desktop version.  only the navigation menu is supposed to be.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193679333-669f13b9-08c6-4e5d-9141-574e2ac9fb63.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193679804-08755727-1a2f-4879-affa-560d81ae587b.png)